scripts/preprocess/process_counties_to_states.js

```diff
@@ -5,26 +5,60 @@
 // in the system() call in process_counties_to_states().)
 console.log("NODE_PATH: " + process.env.NODE_PATH);
 
-// need to run npm install -g topojson geojson yargs if you haven't already
+// you need to run "npm i -g topojson geojson d3 yargs" if you haven't already
 var fs = require('fs');
 var topojson = require('topojson');
 var geojson = require('geojson');
+var d3 = require('d3');
 const argv = require('yargs').argv;
 
 // read in the topojson file with county boundaries and the json dictionary of state names and IDs
 var countyTopo = JSON.parse(fs.readFileSync(argv.counties, 'utf8'));
 var stateDict = JSON.parse(fs.readFileSync(argv.statedict, 'utf8'));
 
+// load the projection we'll be using in the main vizzy.
+// note that we're using a copy of custom_project.js. this is because we needed to reformat
+// into a module to be able to use node's require() function to load the script. I tried
+// jquery.getScript() and failed; require() works great except you need the module formatting.
+var proj = require('./custom_projection.js');
+
+// define the projection. this code is copied from build_map.js
+var waterUseViz = {
+  dims: {
+    map: {
+      width: 1000,
+      height: 700
+    }
+  }
+};
+var projection = proj.albersUsaTerritories()
+  .scale([1200])
+  .translate([waterUseViz.dims.map.width / 2, waterUseViz.dims.map.height / 2]);
+  // default is .rotate([96,0]) to center on US (we want this)
+var buildPath = d3.geoPath()
+  .projection(projection);
```

Comment:
Code from line 26 to here is copied from build_map.js; as with custom_projection.js, I think there might be ways to copy less code, but hey, it's Monday night for a viz due Wednesday, so I propose that we settle for having a comment about this copying (line 25).